api/client/utils.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 var (
-	errConnectionRefused = errors.New("Cannot connect to the Docker daemon. Is 'docker -d' running on this host?")
+	errConnectionRefused = errors.New("Cannot connect to the Docker daemon. Is the docker daemon running on this host?")
```

Comment:
I'm being pedantic, but maybe it's better to call this `errConnectionFailed`. Since this error is also returned in a timeout condition, it doesn't necessarily indicate a refusal.